packages/lune/tsconfig.json

```diff
@@ -3,10 +3,11 @@
   "compilerOptions": {
     "target": "ESNext",
     "module": "ESNext",
-    "moduleResolution": "node",
+    "moduleResolution": "Node",
     "declaration": true,
     "outDir": "dist",
     "allowSyntheticDefaultImports": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "skipLibCheck": true
```

Comment:
what does this do?

Reply:
make typescript not angy about farm when bundling
unfortunately despite spending 4 and a half hours on attempting to get it to bundle it no worky so i invite you to do it

Reply:
fine ig